### PR TITLE
Using textures in Editor

### PR DESCRIPTION
**Pull request by lukasesser**
Textures can be used by adding the URL as a comment next to the texture uniform

https://github.com/patriciogonzalezvivo/glslEditor/issues/29#issuecomment-361668953

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
         <a href = "shaders/index.html?data=template.frag">Test Red</a>
       </div>
       <div class="col">
-        <a href = "shaders/index.html?data=texture_test.frag">Texture Test</a>
```
Comment (edap) [inline index.html]: If you have tested it and it works, lets use a name that does not contain the word test, but that tells that this example is using a texture. Maybe template_texture.frag?
Is it necessary that the the .frag file lies in the same folder where the jpg image is? if a new folder creation is necessary, same as above, i would not called it "test" ;)

**Comment by lukasesser** [comment]
tested it and it works – updated the names and I think it makes sense to collect all textures in one subfolder 'textures'

File: shaders/template_texture.frag [hunk 2]
```diff
@@ -5,7 +5,7 @@ precision mediump float;
 
 uniform float u_time;
 uniform vec2 u_resolution;
-uniform sampler2D u_tex0; //texture_test/ko8y68wfsk251.jpg
+uniform sampler2D u_tex0; //textures/ko8y68wfsk251.jpg
```
Comment (edap) [inline shaders/template_texture.frag:8]: yes, sounds good to me ;)

**Review by edap — APPROVED** [review]
It looks good to me.

**Pull request merged**